compile-release-tools

```diff
@@ -23,6 +23,7 @@ RELEASE_TOOLS=(
   gh2gcs
   kubepkg
   krel
+  scheduler-builder
```

Comment:
```suggestion
  schedule-builder
```

All references should updated to be `schedule-builder`, instead of `scheduler-builder`.